rss-client/src/app/services/cheater.service.ts

```diff
@@ -0,0 +1,74 @@
+import { Injectable } from '@angular/core';
+import { Subject } from 'rxjs';
+
+
+/** CheaterService:
+ *  this service is here to track the user's mouse movements,
+ *  as well as certain keypresses and if they switch tabs.
+ *  This is designed to be combined with our 'Quiz' Component,
+ *  so that if the user tries to leave the page they'll get a
+ *  warning. If they continue, their quiz will be void.
+ */
+@Injectable({
+  providedIn: 'root'
+})
+export class CheaterService {
+
+  hidden: string;
+  visibilityChange: string;
+  confirmMessage: string;
```

Comment:
the variables hidden, visibilityChange, and confirmMessageare declared but not used